docs/configuration/configuration.md

```diff
@@ -451,6 +451,11 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Limit on bucket factor of a single native histogram.
+# If this is exceeded, native histogram will be automatically reduced.
+# failed. 1.00271 means the minimum factor supported in prometheus.
```

Comment:
Please add conversion table from factor to schema/scale as explained in the general conversation.

Reply:
We can use round numbers in the table as we just have to hit the right band.

For example, the actual factor for schema 1 is 1.4142..., for 2 it is 1.1892, for 3 it is 1.0905..., for 4 it is 1.0443...

In the table we could say something like this:

```
min factor | resulting schema AKA scale
-----------+------------------
   2       |    0
   1.4     |    1
   1.2     |    2
   1.1     |    3
   1.05    |    4
```

Reply:
About the wording: I would write something like:

"Lower limit for the growth factor of one bucket to the next in each native histogram. The resolution of a histogram with a lower growth factor will be reduced until it is within the limit."

Reply:
@beorn7 @krajorama I added the conversion table and modify the comments as you suggested.